R/chrome.R

```diff
@@ -87,7 +87,14 @@ find_chrome <- function() {
     } else if (is_linux()) {
```

Comment:
Can you just change this to the following?

```suggestion
    } else if (is_linux() || is_openbsd()) {
```